lib/matplotlib/offsetbox.py

```diff
@@ -42,6 +42,20 @@
 DEBUG = False
 
 
+def _is_vector_renderer(renderer):
+    # 1. Check the renderer name
+    vector_bases = {'RendererPdf', 'RendererSVG', 'RendererPS'}
+    # If it's MixedModeRenderer, get the actual renderer recursively
+    actual_renderer = renderer
+    while type(actual_renderer).__name__ == 'MixedModeRenderer':
+        actual_renderer = getattr(actual_renderer, '_renderer', actual_renderer)
+    renderer_name = type(actual_renderer).__name__
```

Comment:
This approach feels quite hacky. The functionality should live in a method `MxedModeRenderer.is_vector_renderer()`. Possibly that function should be made a core feature `Renderer`. Though I'm not sure whether there are implications for third party renders. @anntzer what do you as a backend implementer think?